src/models/album.rs

```diff
@@ -26,6 +26,17 @@ impl Album {
         album::table.filter(album::id.eq(id)).first::<Self>(conn)
     }
 
+    pub fn get_artwork_url(id: &str) -> String {
+        format!("/files/artwork/{}/raw", id)
+    }
+
+    pub fn artwork_url(&self) -> Option<String> {
+        match self.artwork_path {
+            Some(..) => Some(Album::get_artwork_url(&self.id.to_string())),
```

Comment:
Why do you use `..` here instead of `_` like below?

Reply:
Because it looks cool.